pages/guides/python/flask.zh-TW.mdx

```diff
@@ -5,4 +5,36 @@ ogImageSubtitle: 將你的 Flask 專案部署到 Zeabur
 
 # 部署 Flask 專案
 
-本頁面尚未完成，如果你願意幫助 Zeabur 完善本頁面，讓更多開發者可以快速部署他們的專案，請提交 Pull Request 到 [zeabur/zeabur-docs](https://github.com/zeabur/docs)。
+這篇文章將會教你如何在 Zeabur 一鍵部署 Flask 應用程式。
+
+## 1. 創建 Flask 專案
+
+你可以在 [flask-template](https://github.com/zeabur/flask-template) 查看 Flask 的示例。示例專案非常簡單，包含了一個返回字符串的路由。
+
+```python 
+from flask import Flask
+import os
+
+app = Flask(__name__)
+
+@app.route('/')
+def index():
+    return "Welcome, this is a Flask app deployed on Zeabur"
+
+if __name__ == '__main__':
+    app.run(debug=True, port=os.getenv("PORT", default=5000), host='0.0.0.0')
+```
+
+請註意，Zeabur 使用環境變量 `PORT` 來獲取需要轉發的端口號。因此你需要設置這一環境變量，並在應用中使用 `os.getenv` 或其他方式來設置端口號。默認情況下，Zeabur 將使用 5000 端口。
```

Comment:
Port 5000 is for local development. For services deploying on Zeabur, we need to use the environment variable.